core/domain/event_services.py

```diff
@@ -23,8 +23,8 @@
 from core import jobs_registry
 from core.domain import exp_domain
 from core.platform import models
-(stats_models,feedback_models) = models.Registry.import_models([
-    models.NAMES.statistics, models.NAMES.feedback])
+(stats_models,feedback_models,user_models) = models.Registry.import_models([
```

Comment:
There should be spaces after each comma. (Sorry, I realize they weren't there before.)